src/model/formulations/Gar1962/status.jl

```diff
@@ -2,28 +2,83 @@
 # Copyright (C) 2020, UChicago Argonne, LLC. All rights reserved.
 # Released under the modified BSD license. See COPYING.md for more details.
 
+"""
+    _add_status_vars!
+
+Create `is_on`, `switch_on`, and `switch_off` variables.
+Fix variables if a certain generator _must_ run or based on initial conditions.
+"""
 function _add_status_vars!(
     model::JuMP.Model,
     g::Unit,
     formulation_status_vars::Gar1962.StatusVars,
+    ALWAYS_CREATE_VARS = false
```

Comment:
This could be a parameger of `Gar1962.StatusVars`. More specifically, we could modify `Gar1962.StatusVars` as follows:

```julia
struct StatusVars <: StatusVarsFormulation
    always_create_vars::Bool

    StatusVars() = new(false)
end
```